Space-Zoologist/Assets/DialogueEditor/Assets/Scripts/UI/ConversationManager.cs

```diff
@@ -710,5 +710,15 @@ public void ToggleBacklog()
             bool active = BacklogGameObject.activeSelf;
             BacklogGameObject.SetActive(!active); // toggle backlog
         }
+        Button pingTarget;
+        public void AskForOneTimePing(Button target) {

```

Comment:
During the tutorial, the NPC disappeared while instructing the player to perform actions in game. I'm thinking the problem might be one of these methods, please submit a new pull request if you get it figured out.

Reply:
This is actually something intended. This acts as a temporary solution for instructing the player to perform an action. The player proceeds in the dialogue after they performed the action. Perhaps we should look for another way to do this (like highlighting)?